tests/acceptance/computed-styles-test.js

```diff
@@ -0,0 +1,52 @@
+import Ember from 'ember';
+import {
+  module,
+  test
+} from 'qunit';
+import startApp from '../helpers/start-app';
+import Environment from 'dummy/config/environment';
+
+var application;
+
+module('Acceptance - Computed Styles', {
+  beforeEach: function() {
+    application = startApp();
+    application.registry.optionsForType('expectation', { instantiate: false });
+  },
+
+  afterEach: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+
+Environment.ACCEPTANCE_TESTS.forEach(function(name) {
+  test(name, function(assert) {
+    visit('/acceptance/' + name);
+
+    andThen(function() {
+      var $root = Ember.$(application.rootElement);
+
+      // Look up the expected styles for this test
+      var expectation = application.registry.lookup('expectation:acceptance/tests/' + name);
+      var selectors = Object.keys(expectation.styles);
+
+      // Locate each element with an expected set of styles, and assert that it matches those styles
+      selectors.forEach(function(selector) {
+        var expectedStyles = expectation.styles[selector];
+        var selectorMatches = $root.find(selector);
+
+        // Ensure we're only working against a single element
+        if (selectorMatches.length !== 1) {
```

Comment:
It felt a little strange to me to have an test assertion that was really verifying the setup rather than any kind of functionality of the addon. Don't feel strongly either way though.